vnext/DeforkingPatches/ReactCommon/yoga/yoga/Yoga.cpp

```diff
@@ -765,126 +822,141 @@ float YGNodeStyleGetBorder(const YGNodeConstRef node, const YGEdge edge) {
 // Yoga specific properties, not compatible with flexbox specification
 
 // TODO(T26792433): Change the API to accept YGFloatOptional.
-float YGNodeStyleGetAspectRatio(const YGNodeConstRef node) {
+YOGA_EXPORT float YGNodeStyleGetAspectRatio(const YGNodeConstRef node) {
```

Comment:
>YOGA_EXPORT [](start = 0, length = 11)

FYI @vmoroz, if we need ABI-safe Yoga we can seemingly set this macro and add a calling convention now. I think the actual APIs might already only be using C style structs.